.github/stale.yml

```diff
@@ -1,12 +1,11 @@
 only: issues
-daysUntilStale: 30
+daysUntilStale: 14
```

Comment:
(non-blocking) (just question)
Why 14 days?

Reply:
I wanted to keep things current and focused, hence the shorter amount. 

But I can see how it might be too aggressive. I reverted to `30`.